eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -232,151 +230,4 @@ function Upload-Blobs
-            # If javadocs are produced for a library with source, there will always be an
-            # index.html. If this file doesn't exist in the UnjarredDocumentationPath then
-            # this is a sourceless library which means there are no javadocs and nothing
-            # should be uploaded to blob storage.
-            $IndexHtml = Join-Path -Path $UnjarredDocumentationPath -ChildPath "index.html"
-            if (!(Test-Path -path $IndexHtml))
-            {
-                Write-Host "$($PkgName) does not have an index.html file, skippping."
-                continue
-            }
-
-            # Get the POM file for the artifact we're processing
-            $PomFile = $Item.FullName.Substring(0,$Item.FullName.LastIndexOf(("-javadoc.jar"))) + ".pom"
-            Write-Host "PomFile $($PomFile)"
-
-            # Pull the version from the POM
-            [xml]$PomXml = Get-Content $PomFile
-            $Version = $PomXml.project.version
-            $ArtifactId = $PomXml.project.artifactId
-
-            Write-Host "Start Upload for $($PkgName)/$($Version)"
-            Write-Host "DocDir $($UnjarredDocumentationPath)"
-            Write-Host "PkgName $($ArtifactId)"
-            Write-Host "DocVersion $($Version)"
-            $releaseTag = RetrieveReleaseTag "Maven" $PublicArtifactLocation 
-            Upload-Blobs -DocDir $UnjarredDocumentationPath -PkgName $ArtifactId -DocVersion $Version -ReleaseTag $releaseTag
-
-        } Finally {
-            if (![string]::IsNullOrEmpty($UnjarredDocumentationPath)) {
-                if (Test-Path -Path $UnjarredDocumentationPath) {
-                    Write-Host "Cleaning up $UnjarredDocumentationPath"
-                    Remove-Item -Recurse -Force $UnjarredDocumentationPath
-                }
-            }
-        }
-    }
-}
-
-if ($Language -eq "c")
-{
-    # The documentation publishing process for C differs from the other
-    # langauges in this file because this script is invoked for the whole SDK
-    # publishing. It is not, for example, invoked once per service publishing.
-    # There is a similar situation for other langauge publishing steps above...
-    # Those loops are left over from previous versions of this script which were
-    # used to publish multiple docs packages in a single invocation.
-    $pkgInfo = Get-Content $DocLocation/package-info.json | ConvertFrom-Json
-    $releaseTag = RetrieveReleaseTag "C" $PublicArtifactLocation 
-    Upload-Blobs -DocDir $DocLocation -PkgName 'docs' -DocVersion $pkgInfo.version -ReleaseTag $releaseTag
-}
-
-if ($Language -eq "cpp")
-{
-    $packageInfo = (Get-Content (Join-Path $DocLocation 'package-info.json') | ConvertFrom-Json)
-    $releaseTag = RetrieveReleaseTag "CPP" $PublicArtifactLocation
-    Upload-Blobs -DocDir $DocLocation -PkgName $packageInfo.name -DocVersion $packageInfo.version -ReleaseTag $releaseTag
-}
+&$PublishGithubIODocsFn -DocLocation $DocLocation -PublicArtifactLocation $PublicArtifactLocation
```

Comment:
It might be worth testing to check that this function exists and if not give a nice error message. That will help on-boarding of new languages.